src/imgs/ppu_modes_timing.svg

```diff
@@ -1,152 +1,307 @@
-	<rect class="scanline m2" y="260" x="60" width="80"/>
-	<rect class="scanline m3" y="260" x="145" width="172"/>
-	<rect class="scanline m30" y="260" x="322" width="112"/>
-	<rect class="scanline m0" y="260" x="439" width="87"/>
-	<text class="middle-baseline small" x="535" y="272">...</text>
-	<text class="middle-baseline right" x="55" y="297">3</text>
-	<rect class="scanline m2" y="285" x="60" width="80"/>
-	<rect class="scanline m3" y="285" x="145" width="172"/>
-	<rect class="scanline m30" y="285" x="322" width="112"/>
-	<rect class="scanline m0" y="285" x="439" width="87"/>
-	<text class="middle-baseline small" x="535" y="297">...</text>
-	<text class="middle-baseline right" x="55" y="322">4</text>
-	<rect class="scanline m2" y="310" x="60" width="80"/>
-	<rect class="square m3" y="315" x="150"/>
-	<rect class="square m3" y="315" x="175"/>
-	<rect class="square m3" y="315" x="200"/>
-	<text class="middle-baseline right" x="55" y="347">...</text>
-	<rect class="square m2" y="340" x="70"/>
-	<rect class="square m2" y="340" x="95"/>
-	<rect class="square m2" y="340" x="120"/>
-	<text class="middle-baseline right" x="55" y="372">...</text>
-	<rect class="square m2" y="365" x="70"/>
-	<text class="middle-baseline right" x="55" y="397">...</text>
-	<rect class="square m2" y="390" x="70"/>
-	<text class="middle-baseline right" x="55" y="422">143</text>
-	<rect class="scanline m2" y="410" x="60" width="80"/>
-	<rect class="scanline m3" y="410" x="145" width="172"/>
-	<rect class="scanline m30" y="410" x="322" width="112"/>
-	<rect class="scanline m0" y="410" x="439" width="87"/>
-	<text class="middle-baseline right" x="55" y="447">144</text>
-	<rect class="scanline m1" y="435" x="60" width="466"/>
-	<text class="middle-baseline right" x="55" y="472">145</text>
-	<rect class="scanline m1" y="460" x="60" width="466"/>
-	<rect class="square m1" y="490" x="70"/>
-	<rect class="square m1" y="515" x="70"/>
-	<rect class="square m1" y="540" x="70"/>
-	<text class="middle-baseline right" x="55" y="572">153</text>
-	<rect class="scanline m1" y="560" x="60" width="466"/>
-	<rect class="m1" y="435" x="533" height="145" width="2"/>
-	<text class="middle-baseline small" x="540" y="490">10 "scanlines"</text>
-	<text class="middle-baseline small" x="540" y="510">4560 dots (= 1140 cycles)</text>
-	<!-- Totals -->
-	<text class="centered" y="20" x="613">One frame:</text>
-	<text class="centered" y="45" x="613">70224 dots</text>
-	<text class="centered" y="70" x="613">(= 17556 cycles)</text>
-	<text class="centered" y="95" x="613">@ 59.7 fps</text>
-</svg>
+<?xml version="1.0" encoding="UTF-8" standalone="no" ?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" version="1.1" width="638" height="580" viewBox="0 0 638 580" xml:space="preserve">
+<desc>Created with Fabric.js 3.6.3</desc>
```

Comment:
The SVG being redone from scratch causes some issues:

- [ ] Changes are quite hard to trace (`git blame`...); I cannot pinpoint what changes were made to the SVG to improve accuracy. What are they?
- [ ] A lot of properties are now duplicated all over the place, which increases the file size. (I'm not sure what the point of the `<g transform="">`s are, either.)
- [ ] The used fonts are now inconsistent with the rest of the document (Open Sans and Times New Roman vs. Inter):
  [Light-mode render](https://user-images.githubusercontent.com/15271137/133169039-7efd96fe-340d-4886-886a-4601451eecce.png)
- [ ] It also breaks theme integration, most egregiously on the dark "Ayu" mode:
  [Dark-mode render](https://user-images.githubusercontent.com/15271137/133168872-13474d8f-31ea-470b-9957-8e8522104e9b.png)

For these reasons, I think it would be best to port the changes mentioned in bullet 1 to the existing document.
(You don't have to do it yourself, if you don't want to edit the SVG manually I can do so myself.)

If you think it's necessary, I can also add more comments to the SVG, being the original author and all that :P

Reply:
Ah great! I was gonna see myself on my machine what would happen but I wasn't able to get mdbook onto my machine. The only changes I made was remove the (= cycles) comments scattered around the SVG in favour of explaining that a dot is the same as a T-cycle and so to remove ambiguity, I decided it was best not to include the amounts of cycles passed when specifying dots count for each mode. Because the cycles were in M-cycles when, just before, a box of text said any further usage of "cycle/s" on the page will be in reference to T-cycles. 
I could try to manually edit the SVG myself as well, but I'm not able to locally host the website to try it out so perhaps it's best if you do so.

Reply:
I'm rather new to this entire github pull thing so I hope I am not causing any issues! Haha. Anyway, I manually edited the SVG myself. Of course feel free to only apply the changes if you like them! I just found myself confused reading the docs that cycles were specified in M-cycles rather in T-cycles as was specified above. Thank you so much for your work!

Reply:
If anything is

Reply:
If anything is using T-states, they should be explicitly called T-states, not anything else.

Reply:
Well, T-cycles, but I agree.

Reply:
> If anything is using T-states, they should be explicitly called T-states, not anything else.

Isn't "T-state" the official name used in documentation?

Reply:
Yes, but we don't use official names when they are confusing. Otherwise we'd be talking about "characters", not "tiles".